test/functional/test_framework/authproxy.py

```diff
@@ -197,7 +202,7 @@ def _get_response(self):
         response = json.loads(responsedata, parse_float=decimal.Decimal)
         elapsed = time.time() - req_start_time
         if "error" in response and response["error"] is None:
-            log.debug("<-%s- [%.6f] %s" % (response["id"], elapsed, json.dumps(response["result"], default=serialization_fallback, ensure_ascii=self.ensure_ascii)))
+            log.debug("<-%s- [%.6f] %s" % (response["id"], elapsed, self._json_dumps(response["result"])))
         else:
             log.debug("<-- [%.6f] %s" % (elapsed, responsedata))
```

Comment:
The RPC request seems to be logged with the `AuthServiceProxy.__id_count` but the corresponding non erroneous response doesn't seem to be. @maflcko Do you know the reason for it or can we improve it later?

```
-30-> getreceivedbyaddress
<-- [0.000553] {"jsonrpc":"2.0"...
```

Reply:
Also it would be easier on the eyes to log  the suffix `s` after the elapsed time but something maybe for another PR.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/31955#discussion_r1977508527

> The RPC request seems to be logged with the `AuthServiceProxy.__id_count` but the corresponding non erroneous response doesn't seem to be. @maflcko Do you know the reason for it and should we improve it later?

It looks like `response["id"]` is explicitly logged in the v1 case and `responsedata` which includes the `"id"` field is logged in the v2/error case, so the id should be present in both cases. Probably output could be improved and make more consistent though.